boards/px4/fmu-v2/default.cmake

```diff
@@ -10,7 +10,7 @@ px4_add_board(
 	BOOTLOADER ${PX4_SOURCE_DIR}/ROMFS/px4fmu_common/extras/px4fmuv3_bl.bin
 	IO px4_io-v2_default
 	#TESTING
-	#CONSTRAINED_FLASH
+	CONSTRAINED_FLASH
```

Comment:
What's the effect of that?

Reply:
Silencing some of the bootup messages related to serial. I was also thinking of using it to hide things like the status helper of many drivers and modules printing the contents of orb messages.